com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 Version Updated
```

Comment:
You need to remove this section when you add an entry for the first time.

Reply:
I merged it with the release branch where they did it correctly.
I have then added my change to the entry as usual.